doc/release-process.md

```diff
@@ -3,7 +3,7 @@ Release Process
 
 Before every release candidate:
 
-* Update translations (ping wumpus on IRC) see [translation_process.md](https://github.com/digibyte/digibyte/blob/master/doc/translation_process.md#synchronising-translations).
+* Update translations see [translation_process.md](https://github.com/digibyte/digibyte/blob/master/doc/translation_process.md#synchronising-translations).
```

Comment:
This should probably be updated to reflect the DigiByte-Core org repo

Reply:
utACK

Reply:
Good catch @digicontributer ,

I will open a separate PR for this.